mtp_noms_ops/templates/security/credits_history_list.html

```diff
@@ -0,0 +1,101 @@
+<header>
+    <h1 class="heading-xlarge">{{ view.title }}</h1>
+</header>
+
+<aside class="mtp-object-list-toggle print-hidden">
+    <a href="{% url 'security:check_list' %}" class="button-secondary">{% trans 'View pending' %}</a>
+</aside>
+
+{% include 'mtp_common/includes/message_box.html' %}
+
+<p class="font-large">
+    {% blocktrans trimmed count count=form.total_count with number=form.total_count|separate_thousands %}
+    {{ number }} credit
+    {% plural %}
+    {{ number }} credits
+    {% endblocktrans %}
+</p>
+
+<div class="mtp-results-list mtp-check-list">
+<table class="mtp-table table-font-xsmall">
+    <thead>
+        <tr>
+            <th>{% trans 'Date sent' %}</th>
+            <th>{% trans 'Debit card (from)' %}</th>
+            <th>{% trans 'Prisoner (to)' %}</th>
+            <th>{% trans 'Amount' %}</th>
+            <th>{% trans 'Decision' %}</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for check in objects %}
+            <tr class="mtp-check-row">
+                <td>
+                    {{ check.credit.started_at|date:'SHORT_DATETIME_FORMAT' }}
+                </td>
+                <td>
+                    <a href="{{ check.credit|sender_profile_search_url }}" class="mtp-print-url-hidden">{{ check.credit.sender_name }}</a>
+                    <br/>
+                    {{ check.credit|format_card_number }}
+                    &nbsp;
+                    {{ check.credit.card_expiry_date }}
+                    <br/>
+                    {{ check.credit.sender_email }}
+                    <br/>
+                    {{ check.credit.billing_address.postcode }}
+                    <br/>
+                    <br/>
+                    <strong>{% trans 'Rule:' %}</strong>
+                    {{ check.description }}
+                    <br/>
+                    <strong>{% trans 'Decision details:' %}</strong>
+                    {{ check.decision_reason }}
+                </td>
+                <td>
+                    {{ check.credit.prisoner_number }}
+                    <br/>
+                    <a href="{{ check.credit|prisoner_profile_search_url }}" class="mtp-print-url-hidden">{{ check.credit.prisoner_name }}</a>
+                    <br/>
+                    {{ check.credit.prison_name }}
+                </td>
+                <td>
+                    {{ check.credit.amount|currency }}
+                </td>
+                <td>
+                    {% if check.status == 'accepted' %}
+                        <p class="mtp-accepted-tag">{{ check.status|upper }}</p>
+                    {% else %}
+                        <p class="mtp-rejected-tag">{{ check.status|upper }}</p>
+                    {% endif %}
+                    {% trans 'by' %}
+                    {{ check.actioned_by_name }}
+                </td>
+            </tr>
+            {% empty %}
+            <tr>
+                <td colspan="6">{% trans 'There are no credits to check.' %}</td>
```

Comment:
Probably makes more sense to have something like "There are no actioned credits"